pyxdsm/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '2.0.0'
+__version__ = '2.0.4'
```

Comment:
What's the reason for jumping to v2.0.4?

Reply:
Pypi doesn't allow a package with a used version number to be uploaded again. 
So I'm open to changing the version, but since I used some versions while testing (I used test.pypi first, but obviously didn't catch all the errors) we can't use 2.0.0, 2.0.1, 2.0.2, or 2.0.4 anymore

Reply:
Oh I see, that makes sense. Does it just pick off the version from the __init__ file?

Reply:
It uses the version from setup call, which is set based on the __init__  because we set it up that way

Reply:
should we just move to 2.1.0 to get things back on track?

Reply:
Yeah let's do that. I didn't always stick to semantic versioning before (see release 2.0 for example instead of 2.0.0) but this provides good motivation for doing that.